src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,54 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The AddCommand that adds task to the TaskList.
+ *
+ * @author Leong Jia Hao Daniel
+ */
+public class AddCommand extends Command {
+
+    private Task task;
+
+    /**
+     * The constructor for the AddCommand.
+     *
+     * @param task The task that is being added.
+     */
+    public AddCommand(Task task) {
+        this.task = task;
+    }
+
+    /**
+     * Executes the add task command.
+     *
+     * @param ui The ui class which handles the user interface.
+     * @param storage The storage class which deals with the file.
+     * @param taskList The tasklist that stores the tasks.
+     * @throws DukeException throws if there is an error.
```

Comment:
Maybe capitalize the starting of the exception description so it is consistent with the other descriptors?

```suggestion
     * @throws DukeException throws If there is an error.
```